2020/27xxx/CVE-2020-26065.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Expected location (2020/26xxx/CVE-2020-26065.json) of file does not match actual location (2020/27xxx/CVE-2020-26065.json)

Reply:
Typo in a script ;-)